packages/components/src/core/styles/common/defaultTheme.ts

```diff
@@ -418,7 +487,12 @@ export function makeThemeOptions(appTheme: AppTheme): SDSThemeOptions {
   };
 }
 
-const defaultThemeOptions: SDSThemeOptions = makeThemeOptions(defaultAppTheme);
+const chooseTheme = (theme: "light" | "dark") => {
```

Comment:
Is there more than just `light` and `dark`? Should this be an enum with the ability to expand to more than just these two? 🤔

Reply:
Using the MUI type! Done.